app.rb

```diff
@@ -1,8 +1,71 @@
 require 'sinatra/base'
+require './lib/rps_game'
+
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+  
+  enable :sessions
+  
+  get '/' do
+    erb(:index)
+  end
+
+  post '/sign_in' do
+    session[:player1] = params[:player1]
+    session[:player2] = params[:player2]
+    redirect '/play'
+  end
+
+  get '/play' do
+    session[:game] = create_game
+    redirect 'player_1_choice'
+  end
+
+  get '/player_1_choice' do
+    @game = session[:game]
+    erb(:turn1)
+  end
+
+  post '/player_1_choice' do
+    @game = session[:game]
+    session[:player_1_choice] = params[:player_1_choice]
+    redirect '/player_2_choice' if @game.player2 != 'Computer'
+    redirect '/winner'
+  end
+
+  get '/player_2_choice' do
+    @game = session[:game]
+    erb(:turn2)
```

Comment:
As you recognise in your README, it would be great to tighten your code by using the same view for each player's move - a current_turn variable might assist with that.